Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -1144,25 +1144,43 @@ std::optional<NavigatingToAppBoundDomain> WebFrame::isTopFrameNavigatingToAppBou
 }
 #endif
 
-OptionSet<WebCore::AdvancedPrivacyProtections> WebFrame::advancedPrivacyProtections() const
+inline DocumentLoader* WebFrame::policySourceDocumentLoader() const
 {
-    auto* coreFrame = this->coreLocalFrame();
+    auto* coreFrame = coreLocalFrame();
     if (!coreFrame)
-        return { };
+        return nullptr;
 
     auto* document = coreFrame->document();
     if (!document)
-        return { };
+        return nullptr;
 
     auto* topDocumentLoader = document->topDocument().loader();
     if (!topDocumentLoader)
-        return { };
+        return nullptr;
 
     auto* policySourceDocumentLoader = topDocumentLoader;
```

Comment:
I know this was pre-existing, but why don't we just name this `policySourceDocumentLoader` above?